docs/release-notes.md

```diff
@@ -27,6 +24,11 @@ The toolkit enables robust scaling through:
 #### MPS
 - **AllowList:** Allowlist enables administrators to specify which devices (GUIDs) can connect to MPS. Removing "developer mode" in a previous version broke Allowlist functionality. In this release, MPS AllowList functionality allows devices to connect if they are listed in the MPS database and present the proper credentials. If a device either is not listed in the MPS database or has the wrong credentials, MPS will refuse the connection. Along with this change, we have improved the MPS Devices API to make it very easy to add, edit, and remove devices from the MPS database. MPS API documentation can be found out on [Swagger](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/)
 
+#### Sample Web UI
+- **Improvements:** We have made some minor changes and improvements in the Sample Web UI to stream line profile creation and provide additional information on the devices page.
+   * When setting a new random password in the CIRA Config and Profiles the UI now sets the password length for the user, removing the need for the password length field.  The REST APIs still support user defined values from 8 to 32 for password length.
+   * On the devices page, we have added the device Name, GUID, and tags to help with easily finding this information for each device.
```

Comment:
why is Name capitalized? i think not needed.

Reply:
English is hard...